medusa/indexers/indexer_base.py

```diff
@@ -230,8 +230,8 @@ def result(item, threshold, mean):
                 threshold = 1  # Prevent division by zero
             value = item['rating']
             weight = item['ratingcount']
-            res = item['resolution']
-            score = weights.bayesian(weight, value, threshold, mean)
+            res = int(item['resolution'].split('x')[0]) * int(item['resolution'].split('x')[1])
+            score = weights.bayesian(weight * res, value, threshold, mean)
```

Comment:
This is exactly why I quit.  Tired of explaining to deaf ears why code like this is bad.

Reply:
Jip that's me. Mister deaf ears. But at least I finish what i start.

Reply:
Easy to finish garbage.